### PR TITLE
Link reference to github oauth config with jupyter

### PR DESCRIPTION
**Pull request by Christiandike**
add link to the reference documentation for github oauth configuration with jupyterhub. Fix typos

### CODE REVIEW

File: docs/source/reference/config-ghoauth.md [hunk 1]
```diff
@@ -5,15 +5,15 @@ deployment with the following assumptions:
 
 - Running JupyterHub on a single cloud server
 - Using SSL on the standard HTTPS port 443
-- Using GitHub OAuth (using oauthenticator) for login
+- Using GitHub OAuth (using OAuthenticator) for login
```
Comment (GeorgianaElena) [inline docs/source/reference/config-ghoauth.md:8]: Let's add a link to the OAuthenticator docs here also.

```suggestion
- Using GitHub OAuth (using [OAuthenticator](https://oauthenticator.readthedocs.io/en/latest)) for login
```

**Review by GeorgianaElena — COMMENTED** [review]
Thank you @Christiandike, this is great 🌻 ! I left a small suggestions in-line, which you can accept or modify. But this looks good to go.

**Comment by Christiandike** [comment]
Thank you for taking the time to review @GeorgianaElena . I have applied your suggestion

**Comment by GeorgianaElena** [comment]
Thanks @Christiandike 🌻 !

**Pull request merged**